### ISSUE TITLE
Scrape art and metadata from TheGamesDB.net

### ISSUE THREAD

**Issue by smidley** [avjui/Gamez#36]
It would be nice to be able to scrape artwork and metadata for games from http://thegamesdb.net and then store it locally.

Info about  scraping with the API can be found in the wiki here: http://wiki.thegamesdb.net

**Comment by avjui** [comment]
Do you mean that i should replace this for "Update Game List" function?

**Comment by smidley** [comment]
No, I mean that when a game is downloaded, Gamez should query the API for thegamesdb and see if there is available art or metadata to download with it.  If there is, it will be downloaded and placed into the games download folder.

Gamez could also be set up kind of like sickbeard, where you could keep track of what games you have and "post process" them by downloading art and metadata.

**Comment by avjui** [comment]
How must this look like

Gamename/
----------------->Cover.jpg
----------------->fanart/
---------------------------->Picture1
----------------------------->Picture2
----------------->meta.nfo ?

**Comment by smidley** [comment]
Gamename/
---------------->folder.jpg or poster.jpg (front box art)
---------------->fanart.jpg
---------------->banner.jpg
---------------->logo.png
--------->gamename.nfo

**Comment by avjui** [comment]
Please can  you post me a gamename.nfo. So i can see the structur of the file.

**Comment by smidley** [comment]
I don't personally use local nfo files, but I took this info from Advanced Launcher.  Much more information is available from the API, so it could be added also.  This is just the minimal structure.

```
<game>
    <title></title>
    <platform></platform>
    <year></year>
    <publisher></publisher>
    <genre></genre>
    <plot></plot>
</game>
```

**Comment by avjui** [comment]
Hi

I have made some changes and push a developer branch please can you test it?
At the moment it will write only a gamename.nfo into the downloded folder.
!!! Please Note that you must specified your complet download path where SABnzbd downloads your games
This is in Settings -> Downloads -> SABnzbd -> SABnzbd+ Download Folder 

Please give feedback

**Comment by avjui** [comment]
I have a look to the art downloads but I have found some problem. I have seen that some files are png´s and  some are jpg´s. For some art´s we need jpg´s and for some png´s. So i must convert the Images. And thats the problem. I don´t know some small libary to do this and I don´t want some depencies to PIL. Anybody know a small libary for this?

Thanks

**Comment by aaronflorey** [comment]
I tested the development branch, but it wouldn't postprocess the folder at all. I've been trying to work out why, but i can't find it. I'm looking into a python lib to change the images for you.

**Comment by avjui** [comment]
First thanks for testing. Hmm on witch system do you test? Can you post debuglog. Here it work on freenas

**Comment by aaronflorey** [comment]
I'm testing it on Ubuntu server 12.10, with XBOX 360 games. I'll try it again later

**Comment by aaronflorey** [comment]
http://pastebin.com/T8pi5kpK

**Comment by avjui** [comment]
I have made a new issues.
https://github.com/avjui/Gamez/issues/65
